pkgs/dart_mcp_server/README.md

```diff
@@ -119,7 +128,7 @@ file in your workspace:
 }
 ```
 
-To make the Dart MCP server available in every project you open, edit your
+Or, to make the Dart MCP server available in every project you open, edit your
```

Comment:
Is there any value in this section? I think it's a bit confusing because it's somewhat the same as the top option (but probably not the way we'd want people to do it).